src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -285,6 +361,27 @@ void IslandWindow::SetContent(winrt::Windows::UI::Xaml::UIElement content)
     _rootGrid.Children().Append(content);
 }
 
+// Method Description:
+// - Gets the difference between window and client area size.

```

Comment:
Or wouldn't it be better to name this function `GetClient2NonClientSizeDelta`? Because as I read it now it appears to be the size of whole window (with non client area) rather than just the non-client margins summed up. I thing I even named it once that way but changed it for some reason.